src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -0,0 +1,467 @@
+
+<sup>1</sup> For example, for free water in body temperature, the diffusivity in units of microns<sup>2</sup>/ms should be approximately 3.0.
+
+### Directionally-encoded colour (DEC) maps
+
+These maps are saved as 4D NIfTI files with 3 volumes (x,y,z,{RGB}), where the
+three values in each voxel correspond to red, green and blue components indicating
+directionality of the underlying model (each of which must be a non-negative value),
+and the norm of the vector encodes any scalar parameter from the diffusion model.
+Images encoded in this way should include the field "`_desc-DEC`" in order to
+distinguish them from vector-valued maps (see below).
+
+A common example is DEC FA maps, where the fractional anisotropy from the diffusion
+tensor model is coloured according to the direction of the principal eigenvector:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>_model-DTI_desc-DEC_FA.nii[.gz]
+```
+
+### Vector-valued maps
+
+These maps are saved as 4D NIfTI files (x,y,z, 3\*n), where each triplet of volumes
+encodes both a direction in 3D space and some quantitiative value (these are
+sometimes referred to as "fixels"). The number of volumes in the image must be
+3 times the maximal number of fixels that can appear in any voxel in the image
+(since 3 volumes are required to represent the data for each individual fixel).
+For voxels that contain less than this number of fixels, the excess volumes
+should contain either zero-filling, or NaN values. Exactly what is encoded by
+the amplitude of each XYZ triplet depends on the underlying model properties
+that were used to derive the map, with some examples being:
+
+| `<map_label>` values | Description                                                 |
+| -------------------- | ----------------------------------------------------------- |
+| PDF                  | Diffusion propagator                                        |
+| EVECS                | Eigenvectors of a model that has eigenvectors (such as DTI) |
+| PEAKS                | Directions and amplitudes of ODF maxima on the sphere       |
+
+## Describing tractography
+
+Tractography normally generates one of two primary file types: tractograms or
+NIfTi files containing maps. Tractograms are files containing a collection of
+streamlines, which are objects describing the path of idealized brain fiber
+fascicles (ensembles of neural fibers that travel together). Tractography
+visitation maps contain 3D spatial histograms of the number / fraction of
+streamlines intersecting each voxel, but do not contain the individual
+streamlines trajectories themselves. Two tractogram file formats are supported;
+tractography visitation maps should be saved as NIfTIs.
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>[_space-<space>][_desc-<label>][_subset-<label>]_tractography.[trk|tck|nii[.gz]]

```

Comment:
Can all tractography methods be classified as either deterministic or probabilistic? I say that bc for segmentations we have `dseg` and `probseg` suffices